src/salmon/quant/test_data/script.sh

```diff
@@ -0,0 +1,9 @@
+# salmon quant test data
+
+# Test data was obtained from https://github.com/snakemake/snakemake-wrappers/tree/master/bio/salmon/quant/test
+
+if [ ! -d /tmp/snakemake-wrappers ]; then
+  git clone --depth 1 --single-branch --branch master https://github.com/snakemake/snakemake-wrappers /tmp/snakemake-wrappers
+fi
+
+cp -r /tmp/snakemake-wrappers/bio/salmon/quant/test/* src/salmon/quant/test_data
```

Comment:
Can you manually create test files for `salmon_quant` and `salmon_index` to create some smaller fastq files:

- Have much smaller file sizes
- That allows verifying whether the counts resulting from `salmon_quant` make sense?

You can use the code I wrote [here](https://github.com/viash-hub/biobase/blob/main/src/star/star_align_reads/test.sh#L41-L82) as an example.

Reply:
Done!